packages/shipjs/src/step/setup/CI/addGithubActions.js

```diff
@@ -0,0 +1,197 @@
+import { getGitConfig } from 'shipjs-lib';
+import runStep from '../../runStep';
+import fs from 'fs';
+import path from 'path';
+import ejs from 'ejs';
+import mkdirp from 'mkdirp';
+import { print } from '../../../util';
+import { info, warning } from '../../../color';
+
+export default ({
+  baseBranch,
+  releaseBranch,
+  manualPrepare,
+  schedulePrepare,
+  cronExpr,
+  dir,
+  dryRun,
+}) =>
+  runStep(
+    {
+      title: 'Creating GitHub Actions configuration',
+    },
+    () => {
+      const gitUserName = getGitConfig('user.name') || 'Your Name';
+      const gitUserEmail = getGitConfig('user.email') || 'your@email.com';
+
+      const log = [
+        createGithubAction({
+          content: getBaseConfig({ releaseBranch }),
+          actionPath: '.github/workflows/shipjs-trigger.yml',
+          dir,
+          dryRun,
+        }),
+        manualPrepare &&
+          createGithubAction({
+            content: getManualPrepareConfig({
+              baseBranch,
+              gitUserName,
+              gitUserEmail,
+            }),
+            actionPath: '.github/workflows/shipjs-manual-prepare.yml',
+            dir,
+            dryRun,
+          }),
+        schedulePrepare &&
+          createGithubAction({
+            content: getScheduleConfig({
+              baseBranch,
+              cronExpr,
+              gitUserName,
+              gitUserEmail,
+            }),
+            actionPath: '.github/workflows/shipjs-schedules-prepare.yml',
+            dir,
+            dryRun,
+          }),
+
+        () => {
+          print('  You still need to finish setting up at GitHub Actions.');
+          // add link to readme here
```

Comment:
Let's do this in a new PR.
I will open a new PR for the readme and the doc. And there we will add the link here.

Reply:
ok